packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1094,6 +1114,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         this.summarizerClientElectionEnabled = this.isSummarizerClientElectionEnabled();
         this.maxOpsSinceLastSummary = this.getMaxOpsSinceLastSummary();
         this.initialSummarizerDelayMs = this.getInitialSummarizerDelayMs();
+        if (this.runtimeOptions.enableRuntimeIdCompressor) {
+            if (idCompressorSnapshot !== undefined) {
+                this._idCompressor = hasOngoingSession(idCompressorSnapshot) ? IdCompressor.deserialize(idCompressorSnapshot) : IdCompressor.deserialize(idCompressorSnapshot, createSessionId())
+            } else {
+                this._idCompressor = new IdCompressor(createSessionId(), 10, this.logger);
```

Comment:
10 should definitely be either configurable or a constant somewhere :)

Reply:
Is configuring this done commonly? I'm more inclined to make it a constant if not as it's the only configurable part of initializing the compressor. Making this configurable would make enabling it kind of awkward as it would have to be something like `undefined === disabled` and any number specified would be enabled. We could also introduce a new configuration interface but its only member would be the `reservedIdCount`.

Reply:
I agree, a constant factored out to the top of the file or wherever appropriate is fine. 

On a related note, this number can't be changed in a backwards compatible way without a data migration. Should we bump it up to something higher? 100, maybe? 1000? This number represents all the reserved IDs we might ever want for system purposes, or perhaps even DDS purposes if we decide to expose reserved IDs to DDSs. SharedTree needed one, and it seems not unlikely that the framework would want to use some too. @taylorsw04, thoughts? IIRC, bumping this number doesn't cost us any more cluster space or anything like that, but it means that short IDs for small documents will serialize to be larger, i.e. `11, 12, 13, 14, 15` in JSON is fewer characters than `1001, 1002, 1003, 1004, 1005`.